package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@webosose/ares-cli",
-  "version": "2.0.1",
+  "version": "2.0.2",
```

Comment:
We need to discussion. When we update CLI version. 
I think that before release time.

Let's talk.